frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -91,11 +101,25 @@ export const VMResourceSummary: React.FC<VMResourceSummaryProps> = ({
   kindObj,
 }) => {
   const { t } = useTranslation();
+  const templatesResource: WatchK8sResource = {
+    isList: true,
+    optional: true,
+    kind: TemplateModel.kind,
+    namespace: TEMPLATE_VM_COMMON_NAMESPACE,
+    selector: {
+      matchLabels: { [TEMPLATE_TYPE_LABEL]: TEMPLATE_TYPE_BASE },
+    },
+  };
+  const [commonTemplates] = useK8sWatchResource<TemplateKind[]>(templatesResource);
 
   const isVM = kindObj === VirtualMachineModel;
   const vmiLike = isVM ? vm : vmi;
 
-  const template = getVMTemplate(vm, templates);
+  const isCommonTemplate =
+    getLabel(vmiLike, LABEL_USED_TEMPLATE_NAMESPACE) === TEMPLATE_VM_COMMON_NAMESPACE;
+  const template = isCommonTemplate
+    ? getVMCommonTemplate(vmiLike, commonTemplates)
+    : getVMTemplate(vmiLike, templates);
```

Comment:
this change is not needed when you fetch common templates with user templates

Reply:
in the testing environment we have VMs that do not have the template name label when created from common template, so the common template can't use the same method, using the  `getVMCommonTemplate` to catch this cases.